README.md

```diff
@@ -38,6 +38,33 @@ from torchao.quantization.quant_api import (
 quantize_(m, Int4WeightOnlyConfig())
 ```
 
+#### Quantizing for inference with huggingface configs
```

Comment:
for this one, let's just link to huggingface: https://huggingface.co/docs/transformers/main/en/quantization/torchao so we'll have the most up to date version